Project-Descriptions-and-Plans/CV32E4_TBD/Project_Concept_for_CV32E4_TBD_Nov-27_2023.md

```diff
@@ -0,0 +1,250 @@
+# OpenHW Project Concept Proposal: CV32E4_TBD
```

Comment:
I would prefer having a name here and then changing it later so that we can talk about it more agile

Reply:
```suggestion
# OpenHW Project Concept Proposal: CV32E40PX
```